plugins/module_utils/_yaml.py

```diff
@@ -0,0 +1,55 @@
+# PLEASE READ THE FOLLOWING
+# This file is the vendored version of Ansible's `/lib/ansible/module_utils/common/yaml.py`.
+# This file MUST be removed once this collection drops support for ansible-core v2.11, and
+# related imports MUST be changed to `from ansible.module_utils.common.yaml import ...` or
+# alike.
+# This file was taken at https://raw.githubusercontent.com/ansible/ansible/5f6ebebce9f0e7eab1ba26848d2dd02d24f43e0a/lib/ansible/module_utils/common/yaml.py
+
+# (c) 2020 Matt Martz <matt@sivel.net>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
```

Comment:
```suggestion
# A vendored version of ansible-core's `lib/ansible/module_utils/common/yaml.py`.
# This file can be removed once community.docker drops support for ansible-core 2.11.

# Copyright (c) 2020 Matt Martz <matt@sivel.net>
# Simplified BSD License (see LICENSES/BSD-2-Clause.txt or https://opensource.org/licenses/BSD-2-Clause)
# SPDX-License-Identifier: BSD-2-Clause
```
You also need to add https://github.com/ansible/ansible/blob/devel/licenses/simplified_bsd.txt as `LICENSES/BSD-2-Clause.txt`, and add `BSD-2-Clause` to https://github.com/ansible-collections/community.docker/blob/main/galaxy.yml#L15-L17.